static/app/views/issueList/issueListSetAsDefault.tsx

```diff
@@ -47,7 +47,7 @@ function IssueListSetAsDefault({organization, sort, query}: IssueListSetAsDefaul
         normalizeUrl({
           ...location,
           pathname: `/organizations/${organization.slug}/issues/searches/${response.id}/`,
-          query: {referrer: 'search-bar', ...currentQuery},
```

Comment:
The mocks we were providing actually hid this bug. We want the referrer to override the current params.